jq_service_parser_test.go

```diff
@@ -22,6 +22,9 @@ var k8sResource = `{
 			"opslevel.com/lifecycle": "alpha",
 			"opslevel.com/tier": "tier_1",
 			"opslevel.com/product": "jklabs",
+			"opslevel.com/property.my_bool": "{\"owner\": \"platform\", \"value\": true}",
+            "opslevel.com/property.my_string": "{\"owner\": \"leadership\", \"value\": \"hello world\"}",
+            "opslevel.com/property.my_object": "{\"owner\": \"engineering\", \"value\": {\"message\": \"hello world\", \"boolean\": true}}",
```

Comment:
it might help to make these more concrete examples.  each one of these is a single custom property on a service.  We are not targeting multiple services to apply props to.  It should be "all the props" of a single service that should be assigned so `owner` is confusing here.  

If you need help coming up with concrete examples let me know.

Reply:
Re-worked these examples to exclude `owner`.